### DISCUSSION TITLE
eslint rules too strict / different from nextjs

### DISCUSSION THREAD

**Discussion by AaronLayton** [vercel/turborepo#5969] [Help] [answered]
I have created a fresh turborepo and just copying in some components from another NextJS project and it kicks of lots of eslint errors.

![image](https://github.com/vercel/turbo/assets/694964/b2ca691f-d435-4d1f-9799-15664d3a504f)

My understanding was the linting was matching the @vercel standard errors

![image](https://github.com/vercel/turbo/assets/694964/f986f880-674a-454b-a723-0f2c72851ccb)

Is this the expected behaviour and how can I set it back closer to NextJS

**Comment by anthonyshew** [comment, marked as answer]
We've eased up the ESLint configuration in `create-turbo` since the date on your post. The lint configurations you see there come from [our style guide](https://www.npmjs.com/package/@vercel/style-guide#prettier) which we keep pretty tight, not the Next.js default one.

Check out the new `create-turbo` with `npx create-turbo` for a lighter configuration, closer to the defaults in Next.js.
